lib/checkPullRequestLabels.js

```diff
@@ -14,66 +14,6 @@ var matchChangelogLabelWithRegex = function(changelogLabel) {
   return regExpForChangelogLabel.test(changelogLabel);
 };
 
-/**
- * @param {import('probot').Context} context
- */
-module.exports.checkAssignee = async function(context) {
```

Comment:
We will also need to update all the CHANGELOG labels - i.e. remove the project owner names as well as update the check here for verifying if a label is valid (can be done in a separate PR).

Reply:
I'd do it in this PR.

Reply:
Done.

Reply:
When this PR is merged, I'll update the changelog labels